MapleServer2/Servers/Game/PlayerStorage.cs

```diff
@@ -1,32 +1,33 @@
 ﻿using System;
+using System.Collections.Concurrent;
 using System.Collections.Generic;
 using MapleServer2.Types;
 
 namespace MapleServer2.Servers.Game
 {
     public class PlayerStorage
     {
-        private readonly Dictionary<long, Player> IdStorage;
-        private readonly Dictionary<string, Player> NameStorage;
+        private readonly ConcurrentDictionary<long, Player> IdStorage;
```

Comment:
Curious if any other parts of the source need to make this change

Reply:
Hmm, there shouldn't be, I don't think? This is private so it would all be handled in this class, no?